src/components/Office/PPM/ReviewProGear/ReviewProGear.module.scss

```
@@ -50,4 +55,48 @@
   div.reject {
     @include u-margin-top(2);
   }
+
+  :global(.usa-radio__label) {
+    color: $base-darkest;
+    @include u-font-size('body', 'xs');
+  }
+
+  p {
+    color: $base-darkest;
+  }
+
+  .statusOption {
+    :global {
+      .usa-radio__label {
+        font-weight: bold;
+      }
+    }
+  }
+
+
+  :global(.usa-input) {
+      color: $base-darkest;
+      @include u-font-size('body', 'xs');
+  }
+
+  .hint {
+    color: $base-darker;
+    font-size: 13px;
+    line-height: 18px;
+    @include u-margin-top(1);
+    @include u-margin-bottom(0);
+  }
+
+  .acceptRadio {
```

Nit: this could be combined with the `.rejectRadio` rule below to avoid repetition: 

```suggestion
  .acceptRadio, .rejectRadio {
```

---

done